openpower/package/hostboot/Config.in

```diff
@@ -37,3 +37,12 @@ config BR2_HOSTBOOT_CONFIG_FILE
 
 endif
 
+config BR2_HOSTBOOT_USE_ALTERNATE_GCC
+		bool "use Alternate GCC to build hostboot-p8"
```

Comment:
hostboot-p8 is where hostboot for POWER8 lives, not in the hostboot package.

Reply:
Thanks, copy&paste error, since hostboot plain is also covered in here (I couldn't get it to compile with gcc7)